frontend/src/components/PreviewReview/PreviewReview.tsx

```diff
@@ -0,0 +1,174 @@
+import { ChevronLeftIcon, ChevronRightIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  HStack,
+  Icon,
+  Image,
+  Stack,
+  Tag,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import { ReviewResponse } from "../../types/ReviewTypes";
+
+/**
+ * The model defining the props for the Preview Review component
+ */
+interface PreviewReviewProps {
+  /** The review to be previewed */
+  review: ReviewResponse;
+}
+
+/**
+ * This component allows an admin to preview their review and get an understanding of
+ * what the subscribers of the review will see
+ */
+const PreviewReview = ({ review }: PreviewReviewProps): React.ReactElement => {
+  const [currentBook, setCurrentBook] = useState<number>(0);
+
+  const handleLeftClick = () => {
+    if (currentBook > 0) {
+      setCurrentBook(currentBook - 1);
+    }
+  };
+
+  const handleRightClick = () => {
+    if (currentBook < review.books.length - 1) {
+      setCurrentBook(currentBook + 1);
+    }
+  };
+
+  return (
+    <>
+      <VStack spacing={10}>
+        <HStack justify="space-between" mb="-12px">
+          <ChevronLeftIcon
+            boxSize="2em"
+            style={{
+              cursor: `${currentBook !== 0 ? "pointer" : "auto"}`,
+              color: `${currentBook !== 0 ? "black" : "lightgray"}`,
+            }}
+            onClick={handleLeftClick}
+          />
+          <Box paddingRight={16}>
+            <Image
+              borderRadius={5}
+              maxHeight={60}
+              maxWidth={60}
+              objectFit="contain"
+              src={review.books[currentBook].coverImage}
```

Comment:
Could you add nullish checking for `coverImage`? (ie `.coverImage?`) We may/may not make this optional in the future